docs/source/conf.py

```diff
@@ -114,6 +114,25 @@
 # A list of files that should not be packed into the epub file.
 epub_exclude_files = ['search.html']
 
+# Add googleanalytics id
+# ref: https://github.com/sphinx-contrib/googleanalytics/blob/master/sphinxcontrib/googleanalytics.py
```

Comment:
@fg-mindee I have updated this to a newer version.
But any idea how to test with deploy ?

Reply:
seems to be ok
[https://www.googletagmanager.com/gtag/js?id=G-40DVRMX8T4](https://www.googletagmanager.com/gtag/js?id=G-40DVRMX8T4)

Reply:
Not sure how to test it, but worst case scenario, we can revert this

About your last URL, what is this supposed to be? :sweat_smile:

Reply:
we don't have to set `googleanalytics_id` or `googleanalytics_enabled` then?

Reply:
@fg-mindee 
Yes, I would also say that we try it !
We do not need to set googleanalytics_enabled it is only used if you use it as extension but we have integrated it directly so no need to set.
googleanalytics_id is only added to app.config to have a better overview we could set it also directly instead but i would say keep it

Reply:
URL shows that your id is set correctly with its configuration so i think it will work :) 
`{"function":"__gct","vtp_trackingId":"G-40DVRMX8T4","vtp_sessionDuration":0,"tag_id":1}`